src/components/form/form.stories.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Form from './form';
+import Textbox from '../textbox';
+
+storiesOf('Form', module)
+  .add('default', () => {
+    const unsavedWarning = boolean('unsavedWarning', false);
+    const save = boolean('save', true);
+    const cancel = boolean('cancel', true);
+    const buttonAlign = text('buttonAlign', '');
```

Comment:
I think it should be select like on the demo page https://carbon.sage.com/components/form